azure-pipelines.yml

```diff
@@ -0,0 +1,54 @@
+jobs:
+
+- job:
+  strategy:
+    matrix:
+      Mojave-Xcode-10.2.1:
+        IMAGE_POOL: 'macOS-10.14'
+        XCODE_VERSION: '10.2.1'
+      Mojave-Xcode-11.0:
+        IMAGE_POOL: 'macOS-10.14'
+        XCODE_VERSION: '11'
+  pool:
+    vmImage: $(IMAGE_POOL)
+  
+  steps:
+
+  - script: sudo xcode-select --switch "/Applications/Xcode_$(XCODE_VERSION).app/Contents/Developer"
+    displayName: "Select Xcode $(XCODE_VERSION)"
```

Comment:
Let's add the same version and path information as we have in the iOSDeviceManager and DeviceAgent pipelines.

```      set -e
	      sudo xcode-select --switch "/Applications/Xcode_$(XCODE_VERSION).app/Contents/Developer"
	      echo "Xcode version: $(xcrun xcodebuild -version)"
	      echo "Xcode path: $(xcrun -f xcodebuild)"
	      echo "Xcode select path: $(xcode-select --print-path)"
```